app/views/main.scala.html

```diff
@@ -69,7 +80,7 @@
                     </span>
                 </div>
                 <div class="govuk-footer__meta-item">
-                    <a class="govuk-footer__link govuk-footer__copyright-logo" href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/">© Crown copyright</a>
+                    <a class="govuk-footer__link">© Crown copyright</a>
```

Comment:
I'm going to ask Kirsten about the logo issue, but what's the reason for removing the link as well? I would have thought we'd want to keep that to explain what Crown Copyright means?

Reply:
Okay no worries, we can revisit the logo issue after you've spoken with her. And oops I didn't realise I had removed it 🙈  I will add it back